Makefile

```
@@ -116,24 +135,51 @@ TMT_TEST_CONTAINER_IMAGES := $(TMT_TEST_CONTAINER_IMAGE_NAME_PREFIX)/alpine:late
                              $(TMT_TEST_CONTAINER_IMAGE_NAME_PREFIX)/ubuntu/22.04/upstream:latest \
                              $(TMT_TEST_CONTAINER_IMAGE_NAME_PREFIX)/debian/12.7/upstream:latest
 
-TMT_TEST_IMAGES_TARGETS := $(foreach image,$(TMT_TEST_CONTAINER_IMAGES),images-tests/$(subst :,\:,$(image)))
+# The list of targets building individual tmt test images.
+TMT_TEST_IMAGES_TARGETS := $(foreach image,$(TMT_TEST_CONTAINER_IMAGES),images/test/$(subst :,\:,$(image)))
+
+images: $(TMT_DISTRO_IMAGES_TARGETS)  ## Build tmt images for podman/docker
+	podman images | grep 'localhost/$(TMT_DISTRO_CONTAINER_IMAGE_NAME_PREFIX)/' | sort
 
-images-tests: $(TMT_TEST_IMAGES_TARGETS)  ## Build customized images for tests
+images/test: $(TMT_TEST_IMAGES_TARGETS)  ## Build customized images for tests
 	podman images | grep 'localhost/$(TMT_TEST_CONTAINER_IMAGE_NAME_PREFIX)/' | sort
 
-images-tests-bases:  ## Download base images for custom test images
+images/test/bases:  ## Download base images for custom test images
 	podman pull $(TMT_TEST_IMAGE_BASES)
 
+# Build a single container: <image name> <containerfile>
+define do-build-container-image =
+@ echo "$(ccgreen)Building $(ccred)${1}$(ccend) $(ccgreen)image...$(ccend)"
```

```suggestion
@ echo "$(ccgreen)Building$(ccend) $(ccred)${1}$(ccend) $(ccgreen)image...$(ccend)"
```

Should the green part be ended as well?

---

It could be, but it's not necessary because the next character is a white space, and then the red color begins. So, no harm. But it should be, to prevent surprises.

---

I see, there's the same case with the next segment, so I thought it would make sense to have it consistent (either both or none).

---

Fixed in c2fb6108